packages/@uppy/angular/tsconfig.json

```diff
@@ -21,7 +21,10 @@
     "module": "ES2022",
     "useDefineForClassFields": false,
     "lib": ["ES2022", "dom"],
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "paths": {
+      "@uppy/utils": ["../utils/lib/index.d.ts"]
+    }
```

Comment:
paths should never be used from the way I understand it, ideally we find another solution. You can ask Remco in Slack if you aren't sure.